spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -305,6 +307,119 @@
     end
   end
 
+  describe "#list_collections" do
```

Comment:
Something to start considering is how to avoid hitting the database during testing since it drastically slows down the time it takes to run the test suite. For this particular PR it's not a big deal and especially since we are in the process of sunsetting DH, but for future features and refactors on the Galter Website it's something we should strive for. I'll try to put together a little tech talk on it for the future, maybe something for the book club.